PG-autoloads-bootstrap.el

```diff
@@ -0,0 +1,18 @@
+;;;###autoload
```

Comment:
Is ;;;###autoload documented to work before another set of comments? If not, I'd feel safer moving it to right before the function